core/server/master/src/main/java/alluxio/master/file/meta/InodeDirectory.java

```diff
@@ -51,6 +53,12 @@ public Object getFieldValue(Inode<?> o) {
 
   private boolean mDirectChildrenLoaded;
 
+  /**
+   * A null value in mDefaultAcl is used to indicate no default acl set.
```

Comment:
is `null` possible anymore?

Reply:
changed comments to reflect recent change